specs/bellatrix/fork-choice.md

```diff
@@ -72,7 +73,23 @@ As per EIP-3675, before a post-transition block is finalized, `notify_forkchoice
 
 *Note*: Client software MUST call this function to initiate the payload build process to produce the merge transition block; the `head_block_hash` parameter MUST be set to the hash of a terminal PoW block in this case.
 
-*Note*: Until safe head function is implemented, `safe_block_hash` parameter MUST be stubbed with the `head_block_hash` value.
+##### `safe_block_hash`
+
+The `safe_block_hash` parameter in a call to `notify_forkchoice_updated` function
```

Comment:
I'd prefer to write a function wrt the safe beacon block -- `get_safe_beacon_block` -- and then input from the output of that function (`safe_block`) `safe_block.body.execution_payload.block_hash` into the `notify_fcu` function.  This leaves this beacon block based safe function as a first class citizen to be used for any purpose.

Then to encapsulate the additional `Hash32()` logic here we can have `get_safe_execution_payload_hash()` as it's own helper.